xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1210 @@
+     images. The final path will be something like <filename>/tmp/skopeodata/registry.suse.com/</filename>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Populate the local registry with the downloaded data.
+    </para>
+    <para>
+     For <literal>--dest-creds</literal> you must use the credentials you
+     created during <xref linkend="sec.deploy.scenarios.airgap.container-registry.mirror" />.
+    </para>
+<screen>&prompt.user;<command>skopeo sync --dest-creds <replaceable>USERNAME:PASSWORD</replaceable> \
+dir:<replaceable>/tmp/skopeodata/registry.suse.com/</replaceable> <replaceable>docker://myregistry.local:5000</replaceable></command>
+     </screen>
+   </step>
+   <step>
+    <para>
+     After the synchronization is done, you can remove the <filename>skopeodata</filename>
+     directory.
+    </para>
+<screen>&prompt.user;<command>rm -rf /tmp/skopeodata</command></screen>
+   </step>
+   <step>
+    <para>
+     Replace the charts download URL in the index file for the charts
+     repository and rename the file back to <filename>index.yaml</filename>.
+    </para>
+    <screen>
+&prompt.user;<command>cd /tmp/charts</command>
+</screen><screen>
+&prompt.user;<command>sed -i -e 's/https:\/\/kubernetes-charts.suse.com/https:\/\/<replaceable>myregistry.local</replaceable>\/charts/g' \
+downloaded-index.yaml</command>
+</screen><screen>
+&prompt.user;<command>mv downloaded-index.yaml index.yaml</command>
+    </screen>
+   </step>
+  </procedure>
+
+   <procedure>
+    <title>Transfer Data To Secure Storage</title>
+    <step>
+     <para>
+      Connect the trusted storage to the external mirror.
+     </para>
+    </step>
+    <step>
+     <para>
+      Transfer the container image data to the trusted storage. This will remove
+      all files and directories that are no longer present on the external host
+      from the trusted storage.
+     </para>
+<screen>&prompt.user;<command>rsync -aP /tmp/skopeodata/registry.suse.com/ <replaceable>/mnt/storage/container-images/</replaceable> --delete</command>
+     </screen>
+    </step>
+    <step>
+     <para>
+      Transfer the helm chart data to the trusted storage.
+     </para>
+<screen>&prompt.user;<command>rsync -aP <replaceable>/tmp/charts/</replaceable> <replaceable>/mnt/storage/charts</replaceable> --delete</command>
+     </screen>
+    </step>
+   </procedure>
+
+   <procedure>
+    <title>Update Internal Mirror</title>
+    <step>
+     <para>
+      Connect the trusted storage to the internal mirror.
+     </para>
+    </step>
+    <step>
+     <para>
+      Transfer the container image data to the internal mirror. This will remove
+      all files and directories that are no longer present on the trusted storage
+      from the internal mirror. If you have added any images to the registry
+      manually, please back up these first and restore after the sync from
+      the trusted storage is done.
```

Comment:
`If you have added...`: drop that. This internal registry is read-only, there's no way to add stuff to it other than copy low level data straight from the registry storage directory.

Reply:
I expect users to do exactly that. We give them a way to load images from some external registry into their internal one so they will use it for anything they need there. I don't know if we should explicitly clarify that this is not supported.

Reply:
I think this is stated somewhere in the docs (that we don't support that). We told them to make the registry read only, we gave them instructions on how to do that. If they do other things it's not our responsibility. This sentence gives the impression we are going to "tolerate" that.